sdks/python/apache_beam/typehints/native_type_compatibility.py

```diff
@@ -140,6 +154,17 @@ def convert_to_beam_type(typ):
   Raises:
     ~exceptions.ValueError: The type was malformed.
   """
+  if isinstance(typ, typing.TypeVar):
+    # This is a special case, as it's not parameterized by types.
+    # Also, identity must be preserved through conversion.
```

Comment:
I don't understand this comment. Do you mean that the same TypeVar should always convert to the same TypeVariable?

Reply:
Correct.